src/main/java/test/GiftCardDashboardTest.java

```diff
@@ -84,6 +86,86 @@ public void validationMaxGiftCardAmountBlank() {
 		giftCardConfiguration.clickOnClose();
 	}
 	
+	@Test(priority =6, enabled = true, description="Verify validation message of Max Gift Card Amount")
```

Comment:
why test description is same as of above tc

Reply:
I had copied annotation from above test case. I have fixed it according to the respective test case